app/models/domain.rb

```diff
@@ -26,7 +26,7 @@ class Domain < ActiveRecord::Base
 
   accepts_nested_attributes_for :domain_parameters, :allow_destroy => true
   include ParameterValidators
-  validates :name, :presence => true, :uniqueness => true
+  validates :name, :presence => true, :uniqueness => { :case_sensitive => false }
```

Comment:
I added a case-insensitive unique index for this.
I will also attach to this PR another commit making host names case insensitive, feel free to return this PR to waiting on contributor.